phpunit.xml.dist

```diff
@@ -2,7 +2,6 @@
 <phpunit backupGlobals="false"
          backupStaticAttributes="false"
          beStrictAboutTestsThatDoNotTestAnything="false"
-         bootstrap="tests/bootstrap.php"
```

Comment:
Btw, shouldn't this be:

```
 bootstrap="vendor/autoload.php"
```

Reply:
I guess not because all tests pass but I'm a bit surprised this works out of the box 😄

Reply:
`bootstrap="vendor/autoload.php"` can be omitted as, this is the default value.

Reply:
TIL :)